website/docs/reference/resource-properties/constraints.md

```diff
@@ -66,11 +133,18 @@ models:
           - type: foreign_key
             expression: OTHER_MODEL_SCHEMA.OTHER_MODEL_NAME (OTHER_MODEL_COLUMN)
           - type: ...
+
+
 ```
 
 </File>
 
 
+The recommended syntax for defining foreign keys will be included in dbt Core v1.9 (open source) when it is released.
```

Comment:
```suggestion
In dbt Cloud Versionless, you can now use he recommended syntax for defining foreign keys. This will become available in dbt Core v1.9.
```